### PR TITLE
Merge expiration and terms form.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
@johngian r?

File: mozillians/groups/forms.py [hunk 1]
```diff
@@ -82,8 +82,8 @@ class Meta:
         }
 
 
-class GroupInvalidationForm(happyforms.ModelForm):
-    """Model Form for editing access related fields."""
+class GroupTermsForm(happyforms.ModelForm):
```
Comment (johngian) [inline mozillians/groups/forms.py]: Maybe we should change this to ``GroupTermsExpirationForm`` to capture all the functionality.

File: mozillians/groups/forms.py [hunk 2]
```diff
@@ -82,8 +82,8 @@ class Meta:
...
+    """Model Form for handling group terms."""
```
Comment (johngian) [inline mozillians/groups/forms.py]: Same here, please use a more descriptive docstring.

File: mozillians/groups/forms.py (hunk 1 above)
Comment (johngian) [inline mozillians/groups/forms.py]: Lets also change references for var instantiating this form.

**Comment by akatsoulas** [comment]
@johngian pr updated

**Comment by johngian** [comment]
There some stale references to ``GroupTermsForm`` eg
```
mozillians/groups/tests/test_forms.py:        self.validate_group_edit_forms(forms.GroupTermsForm, group, data)
mozillians/groups/tests/test_forms.py:        self.validate_group_edit_forms(forms.GroupTermsForm, group, data)
mozillians/groups/tests/test_forms.py:        self.validate_group_edit_forms(forms.GroupTermsForm, group, data)
mozillians/groups/tests/test_forms.py:        form = self.validate_group_edit_forms(forms.GroupTermsForm, group,
mozillians/groups/tests/test_forms.py:        self.validate_group_edit_forms(forms.GroupTermsForm, group, data)

```

**Comment by akatsoulas** [comment]
I did not `git add` the file :|

**Comment by johngian** [comment]
Tested it locally. Code looks ok R+

**Pull request merged**